### PR TITLE
V1.14 release

### PR DESCRIPTION
**Pull request by plumbis**

### CODE REVIEW

File: content/v1.14/release-notes/1.14.0.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### 🚨 Warnings and breaking changes
+
+* Crossplane Helm images have moved from `index.docker.io` to `xpkg.upbound.io`
+* Changes to TLS certificates. Existing users of external secret stores need to
+  manually update their TLS certificates. Read [Crossplane issue #4565](https://github.com/crossplane/crossplane/pull/4656) for more information. 
+* Removed Vault support for External Secret Stores. Crossplane
+  suggests using the [ESS Plugins] as a replacement.
+* Removed the `controllerConfigRef` from the Configuration package
+  and package revision APIs.
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:31]: I don't think this was removed. CC @turkenh to confirm.

File: content/v1.14/release-notes/1.14.0.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+<!--more--> 
+
+### 🚨 Warnings and breaking changes
+
+* Crossplane Helm images have moved from `index.docker.io` to `xpkg.upbound.io`
+* Changes to TLS certificates. Existing users of external secret stores need to
+  manually update their TLS certificates. Read [Crossplane issue #4565](https://github.com/crossplane/crossplane/pull/4656) for more information. 
+* Removed Vault support for External Secret Stores. Crossplane
+  suggests using the [ESS Plugins] as a replacement.
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:29]: This link is missing a URL.

File: content/v1.14/release-notes/1.14.0.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* New [Crossplane CLI]({{<ref "../cli" >}}) for troubleshooting and package management.
+* Support for ordered deletion with [Usages]({{<ref "../concepts/usages">}}).
+* Alpha support for [real time composition]({{<ref "../concepts/pods#enable-real-time-compositions">}})
+
+
+<!--more--> 
+
+### 🚨 Warnings and breaking changes
+
+* Crossplane Helm images have moved from `index.docker.io` to `xpkg.upbound.io`
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:25]: I believe we still dual publish - we've changed the default to xpkg from index.

File: content/v1.14/release-notes/1.14.0.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+* Removed the `controllerConfigRef` from the Configuration package
+  and package revision APIs. 
+
+### 💡 New features
+
+What's new in v1.14.0
+
+#### Crossplane CLI
+
+Crossplane v1.14 introduces a rebuilt [Crossplane CLI]({{<ref "../cli" >}}).
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:39]: This is probably worth calling out as a "breaking change" above, in that anything that scripted `kubectl crossplane` will need to be updated.

File: content/v1.14/release-notes/1.14.0.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+This release adds the alpha feature "real time compositions" to Crossplane. 
+
+Real time compositions allow Kubernetes to immediately notify Crossplane of
+changes to composed resources. Crossplane doesn't need to poll resources to
+determine their state. 
+
+Real time compositions dramatically speed up the time Crossplane reacts to
+changes or failures of managed resources.
+
+Real time compositions are an alpha feature and isn't enabled by default.
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:59]:
```suggestion
Real time compositions is an alpha feature and isn't enabled by default.
```

We've all agreed to use US English even though it is bad and wrong, right? 😉

File: content/v1.14/release-notes/1.14.0.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+The [Crossplane pods]({{<ref "../concepts/pods#enable-real-time-compositions">}}) 
+documentation has more information about real time compositions. 
+
+#### Ordered resource deletion
+
+Crossplane introduces a new `usages` type to prevent resources from
+deletion or to ensure deletion happens in the correct order. 
+
+Usages defines a dependency mapping between a resource and any dependencies.
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:69]:
```suggestion
A `Usage` defines a dependency mapping between a resource and any dependencies.
```

Perhaps? The kind is singular.

File: content/v1.14/release-notes/1.14.0.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+Usages also allow Crossplane to prevent accidental deletion of a resource.
+
+Usages are an alpha feature and aren't enabled by default.
+
+The [Usages]({{<ref "../concepts/usages">}}) documentation provides information
+on enabling and configuring Usages.
+
+### 🏗️ Notable changes
+
+#### Composition functions
```
Comment (negz) [inline content/v1.14/release-notes/1.14.0.md:82]: Also worth calling out as a warning - the implementation changes completely and anyone using alpha functions in v1.13 will have to stop before updating.

**Comment by negz** [comment]
Cross-reference https://github.com/crossplane/docs/issues/595, which I can fix up once this is merged.

File: content/v1.14/release-notes/1.14.0.md (hunk 1 above)
Comment (turkenh) [reply content/v1.14/release-notes/1.14.0.md:31]: This is correct, we removed it from the Configuration package API in the scope of [this ticket](https://github.com/crossplane/crossplane/issues/4690).
It is worth noting here that this was a no-op before.

Comment (negz) [reply content/v1.14/release-notes/1.14.0.md:31]: Oh, right I missed that this was just talking about Configurations. Makes sense!

File: content/v1.14/release-notes/1.14.0.md (hunk 5 above)
Comment (plumbis) [reply content/v1.14/release-notes/1.14.0.md:59]: I changed this and cheated

> The real time compositions feature is an alpha feature and isn't enabled by default.

**Review by turkenh — APPROVED** [review]

**Pull request merged**